client/components/forms/post-job/3_JobSummary.tsx

```diff
@@ -90,53 +93,49 @@ export default function GigSummary({
             <Heading fontSize="sm">Job Type:</Heading> {jobtype}
           </>
         )}
-      </Container>
-
-      <br />
+      </Stack>
 
       <Container margin="auto" textAlign="center" maxW="100%">
-        <Heading fontSize="lg">Job Details</Heading>
-        <ButtonGray onClick={goToDetails}>Edit</ButtonGray>
+        <Heading fontSize="lg">
+          Job Details <ButtonGray onClick={goToDetails}>Edit</ButtonGray>
+        </Heading>
       </Container>
 
-      <Container maxW="100%">
-        <Heading fontSize="sm">Category:</Heading>
-        <HStack spacing={4} mt="2.5%" mb="2.5%">
-          <Tag size="md" borderRadius="full" variant="solid" color="black">
-            <TagLabel>Keyword</TagLabel>
-          </Tag>
-        </HStack>
+      <Stack spacing={2} p={2} maxW="100%">
+        <Heading fontSize="sm">Keywords:</Heading>
+
+        <KeywordsSection
+          keywordsData={jobKeywords}
+          templateColumns="repeat(3, 3fr)"
+        />
+
+        <br />
```

Comment:
Not sure why, but whenever I see these `<br />` I always chuckle and think about the old school `<blink></blink>` tags. No matter how far we've come in webdev, these bad boys will never die haha.